packages/document/docs/en/fragments/builtin-components.mdx

```diff
@@ -6,26 +6,79 @@ The Badge component is used to display a badge. For example:
 import { Badge } from '@theme';
 
 function App() {
+  // Using text prop
   return <Badge text="info" type="info" />;
+
+  // Using custom children
+  return (
+    <Badge>
+      <img
+        style={{ height: '18px' }}
+        src="https://lf3-static.bytednsdoc.com/obj/eden-cn/uhbfnupenuhf/rspress/rspress-logo.png"
+      />
+      <span>Rspress</span>
+    </Badge>
+  );
 }
 ```
 
 The effect is as follows:
 
 import { Badge } from '@theme';
 
+<Badge text="tip" type="tip" />
 <Badge text="info" type="info" />
 <Badge text="warning" type="warning" />
 <Badge text="danger" type="danger" />
+<Badge text="outlined" outline />
+
+Custom children:
+
+import SearchSvg from '@theme-assets/github';
+
+<Badge type="tip">
+  <img
+    style={{ height: '18px' }}
+    src="https://lf3-static.bytednsdoc.com/obj/eden-cn/uhbfnupenuhf/rspress/rspress-logo.png"
+  />
+  <span>Rspress</span>
+</Badge>
+
+<Badge type="info">
+  <SearchSvg width="16" />
+  <span>Github</span>
+</Badge>
+
+Inlined with text <Badge text="Tip" />
+
+##### H5 <Badge text="Info" type="info" />
+
+#### H4 <Badge text="Warning" type="warning" />
```

Comment:
We may only need H5 for `Inlined with Heading` example, since H4 and H3 will display in Aside component at right side. 
![image](https://github.com/user-attachments/assets/0466da17-ca2c-48eb-9fc0-3cd26fe46fe4)

Reply:
@Timeless0911 How is this a problem? I don’t want to mistake purpose for appearance—the purpose is to faithfully demonstrate that the Badge component can accommodate different levels of headings.

Reply:
Got it, resolved it by no change.